stroeer/web/section/v1/web_section_service.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package stroeer.web.section.v1;
+
+import "stroeer/web/article/v1/web_article.proto";
+
+option java_multiple_files = true;
+option java_package = "de.stroeer.web.section.v1";
+
+service WebSectionService {
+  rpc GetSectionPage (GetSectionPageRequest) returns (GetSectionPageResponse) {}
+}
+
+message GetSectionPageRequest {
```

Comment:
for now, I guess?
Will need to change, as soon as we have stages, with labels, and configs for the page.